.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "rust-analyzer.linkedProjects": ["./examples/demo-rollup/Cargo.toml"]
+}
```

Comment:
In theory, we shouldn't need this config to have rust-analyzer pick up the project. Did you encounter issues without it?

Reply:
Deleted the file